openroad.bzl

```diff
@@ -709,15 +695,17 @@ def build_openroad(
                             ["GDS_ALLOW_EMPTY=(" + "|".join(macros) + ")"] if len(macros) > 0 else [])
     stage_args["generate_abstract"] += ["ABSTRACT_SOURCE=" + abstract_source] if mock_abstract else []
 
-    stage_sources = init_stage_dict(all_stage_names, stage_sources)
+    stage_cfg_sources = init_stage_dict(all_stage_names, {})
     if sdc_constraints != None:
-        stage_sources["synth_sdc"] = [sdc_constraints]
-        stage_sources["clock_period"] = [sdc_constraints]
-    stage_sources["synth"] = list(filter(stage_sources["synth"], lambda s: not s.endswith(".sdc")))
```

Comment:
What does this filtering check achieve? Seems very fickle to me, e.g. if a `filegroup` is referenced.

Reply:
But if `sdc_constraints` is a `filegroup()` like in the `constraints-sram` case (from `BUILD` in this repo), how does the filter achieve anything?

Reply:
And these dependencies being filtered are from the user supplied `stage_sources`, should we alter those?

Reply:
Yes, you're right, it probably a leftover and does nothing right now.